### PR TITLE
Add waterlogging support

### PR DESCRIPTION
**Pull request by williambl**
Reason:
This PR adds waterlogging support to `BlockAnimalSkull` and `BlockHandOfFate`.
#101 can be closed once this PR is merged.

Image of changes:

<sub>Legal Disclaimer:
ANY PULL REQUEST SUBMITTED TO "The Project" (github.com/itsmeow/betteranimalsplus) AND MERGED INTO A RELEASED VERSION OF THE MODIFICATION (content released via CurseForge.com) BECOMES PROPERTY OF "THE OWNER" (its_meow and partner cybercat5555). ALL RIGHTS TO THE SUBMITTED CONTENT IS TRANSFERRED TO THE OWNER. Credit may or may not be given at The Owner's choice.</sub>

### CODE REVIEW

File: src/main/java/its_meow/betteranimalsplus/common/block/BlockAnimalSkull.java [hunk 1]
```diff
@@ -5,15 +5,14 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Maps;
 
-import net.minecraft.block.Block;
-import net.minecraft.block.BlockRenderType;
-import net.minecraft.block.BlockState;
-import net.minecraft.block.ContainerBlock;
-import net.minecraft.block.SoundType;
+import net.minecraft.block.*;
```
Comment (itsmeow) [inline src/main/java/its_meow/betteranimalsplus/common/block/BlockAnimalSkull.java]: Wildcards >:(

File: src/main/java/its_meow/betteranimalsplus/common/block/BlockHandOfFate.java [hunk 2]
```diff
@@ -2,17 +2,16 @@
 
 import its_meow.betteranimalsplus.common.tileentity.TileEntityHandOfFate;
 import its_meow.betteranimalsplus.init.ModItems;
-import net.minecraft.block.Block;
-import net.minecraft.block.BlockRenderType;
-import net.minecraft.block.BlockState;
-import net.minecraft.block.Blocks;
-import net.minecraft.block.HorizontalBlock;
+import net.minecraft.block.*;
```
Comment (itsmeow) [inline src/main/java/its_meow/betteranimalsplus/common/block/BlockHandOfFate.java]: Wildcards part two >:(

File: src/main/java/its_meow/betteranimalsplus/common/block/BlockAnimalSkull.java (hunk 1 above)
Comment (williambl) [reply src/main/java/its_meow/betteranimalsplus/common/block/BlockAnimalSkull.java]: whoops, IDE did that, sorry.

File: src/main/java/its_meow/betteranimalsplus/common/block/BlockHandOfFate.java (hunk 2 above)
Comment (williambl) [reply src/main/java/its_meow/betteranimalsplus/common/block/BlockHandOfFate.java]: IDE >:(

**Pull request merged**